dev-libs/libpreludedb/files/libpreludedb-5.1.0-update_m4_postgresql.patch

```diff
@@ -0,0 +1,544 @@
+--- a/m4/ax_lib_postgresql.m4	2019-09-17 18:49:46.185218603 -0400
++++ b/m4/ax_lib_postgresql.m4	2019-09-22 09:09:22.356777373 -0400
```

Comment:
You don't really need the timestamps here, it doesn't add any value. Please remove them.

Reply:
done